### PR TITLE
Add module recipe-persistence-jdbc

### PR DESCRIPTION
**Pull request by SLytvyn**

### CODE REVIEW

File: recipe-aggregator/src/main/java/com/epm/recipe/aggregator/config/PropertySourcesConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import org.springframework.context.support.PropertySourcesPlaceholderConfigurer;
+import org.springframework.core.io.FileSystemResource;
+
+@Configuration
+public class PropertySourcesConfiguration {
+
+    @Bean
+    public static PropertySourcesPlaceholderConfigurer placeholder(){
+        PropertySourcesPlaceholderConfigurer placeholderConfigurer = new PropertySourcesPlaceholderConfigurer();
+        FileSystemResource fileSystemResource = new FileSystemResource("D:/epam/epm-recipe/recipe-aggregator/build/distributions/recipe-aggregator/recipe-aggregator/conf/db.properties.txt");
```
Comment (iherasymenko) [inline recipe-aggregator/src/main/java/com/epm/recipe/aggregator/config/PropertySourcesConfiguration.java]: Think about the people who do not work at EPAM...

File: recipe-persistence-jdbc/build.gradle [hunk 2]
```diff
@@ -0,0 +1,8 @@
+dependencies {
+    testCompile group: 'junit', name: 'junit', version: '4.12'
```
Comment (iherasymenko) [inline recipe-persistence-jdbc/build.gradle]: I see a dependency, but don't see any tests. What's the point?

File: recipe-persistence-jdbc/build.gradle [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+    compile group: 'com.zaxxer', name: 'HikariCP', version: '3.3.0'
+    compile group: 'org.mariadb.jdbc', name: 'mariadb-java-client', version: '2.4.2'
```
Comment (iherasymenko) [inline recipe-persistence-jdbc/build.gradle]: Try to follow the existing approach of declaring dependencies.

File: build.gradle [hunk 4]
```diff
@@ -20,6 +20,7 @@ subprojects {
         implementation enforcedPlatform('com.fasterxml.jackson:jackson-bom:2.9.9')
         implementation enforcedPlatform('org.thymeleaf:thymeleaf-spring5:3.0.11.RELEASE')
         implementation enforcedPlatform('ch.qos.logback:logback-classic:1.2.3')
+        implementation enforcedPlatform('org.slf4j:slf4j-api:1.7.26')
```
Comment (iherasymenko) [inline build.gradle]: What is the reasoning behind adding this line?

File: recipe-aggregator/src/main/java/com/epm/recipe/aggregator/config/PropertySourcesConfiguration.java (hunk 1 above)
Comment (SLytvyn) [reply recipe-aggregator/src/main/java/com/epm/recipe/aggregator/config/PropertySourcesConfiguration.java]: Will it be better if I use env var?

Comment (iherasymenko) [reply recipe-aggregator/src/main/java/com/epm/recipe/aggregator/config/PropertySourcesConfiguration.java]: Just make the path relative.

File: build.gradle (hunk 4 above)
Comment (SLytvyn) [reply build.gradle]: I thought that hikari need it. Was been removed.

File: recipe-aggregator/src/main/java/com/epm/recipe/aggregator/config/PropertySourcesConfiguration.java (hunk 1 above)
Comment (SLytvyn) [reply recipe-aggregator/src/main/java/com/epm/recipe/aggregator/config/PropertySourcesConfiguration.java]: Update

**Pull request closed without merging**